native/ssz_nif/src/ssz_types/beacon_chain.rs

```diff
@@ -100,13 +100,11 @@ pub(crate) struct HistoricalSummary {
 }
 
 #[derive(Encode, Decode)]
-pub(crate) struct DepositBase<C: Config> {
-    pub(crate) proof: FixedVector<Bytes32, C::DepositContractTreeDepth>,
+pub(crate) struct Deposit {
+    pub(crate) proof: FixedVector<Bytes32, typenum::U33>,
```

Comment:
this is the way lighouse does it, because the number is DepositContractTreeDepth + 1.